distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -1579,6 +1345,9 @@ type Build_Style_Builder
          by the containing invocation of `Vector.build`.  The error is not added
          to the builder.
 
+         If an uncaught dataflow error is appended with `append`, the dataflow
+         error is propagated, and it is not added to the builder.
```

Comment:
Where is the error propagated? This is unclear to me.

Do you mean that the error will be propagated to the result of calling `to_vector`? Is the first such error propagated? Or last one?

Reply:
Or looking at the code it seems that it is raised as a wrapped panic? That should be clarified by the docs. "Propagated" is far too vague here as it has several possible meanings.

Reply:
Good point -- my documentation was written for `build` and didn't cover `new_builder`. I updated it to cover both cases, and added similar tests for `new_builder`. I changed 'propagate' to 'throw' where 'propagate' is confusing.